functests/1_performance/hugepages.go

```diff
@@ -59,10 +59,10 @@ var _ = Describe("[performance]Hugepages", func() {
 				hugepagesSize, err := machineconfig.GetHugepagesSizeKilobytes(page.Size)
 				Expect(err).ToNot(HaveOccurred())
 
-				availableHugepagesFile := fmt.Sprintf("/sys/kernel/mm/hugepages/hugepages-%skB/nr_hugepages", hugepagesSize)
+				availableHugepagesFile := fmt.Sprintf("/sys/devices/system/node/node%d/hugepages/hugepages-%skB/nr_hugepages", *page.Node, hugepagesSize)
```

Comment:
We need to test the case where no numa node is specified. In such case the hugepages are evenly split across all numa nodes.

What I am saying is that I think this will break in such case when running in discovery mode.

Reply:
```
if page.Node == nil {
					continue
				}
```
so it should work as expected